stock_location_empty/models/stock_location.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, fields, models
+
+
+class StockLocation(models.Model):
+    _inherit = 'stock.location'
+
+    stock_amount = fields.Integer(
```

Comment:
I'd prefer stock_quantity as amount is more related to monetary.
Moreover, this field should be Float